src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,27 @@
+package lotto
+
+private const val NUMBERS_SIZE = 6
+
```

Comment:
what's the different between outer class const value and compaion object const value?